src/Main.java

```diff
@@ -1,15 +1,52 @@
+import java.util.Arrays;
+
 //TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
 // click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
     public static void main(String[] args) {
-        //TIP Press <shortcut actionId="ShowIntentionActions"/> with your caret at the highlighted text
-        // to see how IntelliJ IDEA suggests fixing it.
-        System.out.printf("Hello and welcome!");
-
-        for (int i = 1; i <= 5; i++) {
-            //TIP Press <shortcut actionId="Debug"/> to start debugging your code. We have set one <icon src="AllIcons.Debugger.Db_set_breakpoint"/> breakpoint
-            // for you, but you can always add more by pressing <shortcut actionId="ToggleLineBreakpoint"/>.
-            System.out.println("i = " + i);
+        // Task 1
+        System.out.println("Задача 1");
+        int[] payments = {200, 150, 220, 100, 180};
+        int summaPayment = 0;
+        for (int payment : payments) {
+            summaPayment += payment;
+        }
+        System.out.println("Сумма трат за месяц составила " + summaPayment + " рублей");
+        // Task 2
+        System.out.println("Задача 2");
+        int maxPayment = -1;
+        int minPayment = 10000000;
```

Comment:
Если минимальное значение в массиве будет 10000001, то код работать будет некорректно. Обычно за изначальное значение берут первый элемент массива
```suggestion
        int minPayment = payments[0];
```